### ISSUE TITLE
[Request] Attributes Exceptions 

### ISSUE THREAD

**Issue by stavrianos** [praeclarum/sqlite-net#728]
I am wondering if there is a future plan to insert some individuals exceptions for each attribute like Unique. When there is a new no unique value it just raises an exception with message "Constraint".  I would like to have a specific exception so I can handle.

**Comment by coldacid** [comment]
In addition, it'd be good if such exceptions could have details such as which constraint failed.

**Comment by praeclarum** [comment]
Good idea! I am limited by sqlite's error reporting system and am not sure if I can get the exact constraint name, but it's certainly worth looking into.
